### ISSUE TITLE
Stdio or Sockets is better on Sublime?

### ISSUE THREAD

**Issue by PHdrr** [objeck/objeck-lsp#9]
Better performance and reliability? I just bought Sublime. I don't want to fight with these stupid text editors anymore.

**Comment by objeck** [comment]
Ha, Stdio

**Comment by PHdrr** [comment]
> Ha, Stdio

People said stdio is very slow on Windows: https://github.com/prabirshrestha/vim-lsp/issues/469#issuecomment-524005779

**Comment by objeck** [comment]
I have not had any issues with STDIO on Windows. Both STDIO and sockets work just fine. At some point, I will add support for named pipes on Windows.

**Comment by PHdrr** [comment]
OK. I will go with stdio. Thank you.

**Comment by iqskpduswupkcjqg** [comment]
Finally I decided to buy Sublime too. I don't want to wrestle with these `free and open source` text editors anymore. VSCode is having a weird bug with graphics on my computer. It's not quite usable.

**Comment by objeck** [comment]
> Finally I decided to buy Sublime too. I don't want to wrestle with these `free and open source` text editors anymore. VSCode is having a weird bug with graphics on my computer. It's not quite usable.

I like it, however, I need to figure out how to provide "workspace folders" support for projects. Sublime does not support this feature VS Code does. I plan to figure it out over the weekend.

**Comment by objeck** [comment]
I am wrong Sublime does support workspaces the same way that VS Code does. You must do the following 'File' -> 'Open Folder'. If you have a [build.json](https://github.com/objeck/objeck-lsp/blob/main/README.txt) file in the folder, the LSP server will use it. I tried it with Sublime, and it worked without any code changes. 

Now I can work on the GTK code generator using it's build configuration. I played around with this over the weekend and saw workspaces enabled, but I could not figure out how I did it. I even started modifying the LSP server code before reverting the changes assuming it **would** work if the client was configured correctly.
